pytorch_lightning/trainer/trainer.py

```diff
@@ -296,6 +297,10 @@ def __init__(
                 In 'max_size_cycle' mode, the trainer ends one epoch when the largest dataset is traversed,
                 and smaller datasets reload when running out of their data. In 'min_size' mode, all the datasets
                 reload when reaching the minimum length of datasets.
+
+            stochastic_weight_avg: Whether to `Stochastic Weight Averaging (SWA)
+                <https://pytorch.org/blog/pytorch-1.6-now-includes-stochastic-weight-averaging/>_`
```

Comment:
can we rather refer to our docs? as it adds own callback...

Reply:
Yeah, that's better

Reply:
I think a PyTorch tutorial is better and it makes the description shorter..